### PR TITLE
Override self generated cluster common name in helm chart

### PR DESCRIPTION
**Pull request by jfwm2**
- this gives the possibility to specified a custom name
   that is used for naming of apps, sts, pods, etc.

Signed-off-by: Jean-Francois Weber-Marx <jf.webermarx@criteo.com>

### CODE REVIEW

**Comment by jfwm2** [comment]
Hello
I cannot check the details of the since it point to a private ip address (RFC-1918) http://192.168.200.205:8080/job/Aerospike%20Kubernetes%20Operator/job/PR-34/1/display/redirect

File: helm-charts/aerospike-cluster/templates/_helpers.tpl [hunk 1]
```diff
@@ -25,8 +25,12 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 
 {{- define "aerospike-cluster.commonName" -}}
+{{- if .Values.commonName -}}
+{{- .Values.commonName -}}
```
Comment (spkesan) [inline helm-charts/aerospike-cluster/templates/_helpers.tpl:29]: Can you add a commented out entry in `values.yaml` for `commonName` configuration?
and also document the configuration in README file?

**Review by spkesan — CHANGES_REQUESTED** [review]
Looks good to me. Just minor steps before merging the PR.

**Comment by spkesan** [comment]
> Hello
> I cannot check the details of the test build since it points to a private ip address (RFC-1918) http://192.168.200.205:8080/job/Aerospike%20Kubernetes%20Operator/job/PR-34/1/display/redirect

@jfwm2 Also, don't worry about the failed checks now. It is unrelated to the helm chart changes.

File: helm-charts/aerospike-cluster/templates/_helpers.tpl (hunk 1 above)
Comment (jfwm2) [reply helm-charts/aerospike-cluster/templates/_helpers.tpl:29]: Hello @spkesan I made the changes you requested in another commit; if it is ok for you, I will squash everything before the merge.

**Comment by arrowplum** [comment]
@spkesan can you check with @sud82  and update the changes requested and merge?

**Review by spkesan — APPROVED** [review]

**Pull request merged**